src/Wallabag/CoreBundle/Resources/translations/messages.es.yml

```diff
@@ -1,18 +1,18 @@
 security:
     login:
-        page_title: 'Bienvenido a wallabag !'
-        keep_logged_in: 'Mantenme conectado'
-        forgot_password: '¿Ha olvidado su contraseña?'
+        page_title: '¡Bienvenido a wallabag !'
+        keep_logged_in: 'Manténgame conectado'
+        forgot_password: '¿Se ha olvidado de su contraseña?'
         submit: 'Conectarse'
         register: 'Registrarse'
         username: 'Nombre de usuario'
         password: 'Contraseña'
         # cancel: 'Cancel'
     resetting:
-        description: "Introduce tu dirección de email y le enviaremos las instrucciones para resetear la contraseña"
+        description: "Introduzca su dirección del correo electrónico y le enviaremos las instrucciones para reiniciar la contraseña"
     register:
-        # page_title: 'Create an account'
-        # go_to_account: 'Go to your account'
+        # page_title: 'Crear una cuenta'
+        # go_to_account: 'Acceder su cuenta'
```

Comment:
Could you resubmit a PR and remove the "`# `" before that lines to enabled them?
And apply that on the two other lines below (line 28 and line 32).

Thanks